augur/cli/db.py

```diff
@@ -331,3 +332,188 @@ def check_pgpass_credentials(config):
         else:
             logger.info("Credentials found in $HOME/.pgpass")
 
+@cli.command('update-org-repositories')
+@click.argument('org')
+@pass_application
+def update_org_repositories(augur_app, org):
+    """
+    Add new repositories for specific organization to Augur's database
+    """
+
+    repos = get_all_repos(org)
+    repo_group_id = get_org_id(augur_app, org)
+    existing_repos = get_existing_repos(augur_app, repo_group_id)
+
+    for repo in repos:
+        if repo['node']['name'] not in existing_repos:
+            insert_repo(augur_app, repo['node'], org, repo_group_id)
+
+def generate_repo_id(augur_app):
+    repos_sql = s.sql.text("""
+            SELECT repo_id FROM repo
+            ORDER BY repo_id DESC
+        """)
+
+    repo_id = augur_app.database.execute(repos_sql).fetchone()
+    return repo_id[0]+1
+
+def insert_repo(augur_app, repo, org, repo_group_id):
+    logger.info("Inserting repo {}".repo['name'])
+    repo_id = repo_id = generate_repo_id(augur_app)
+    insert_repo_sql = s.sql.text("""
+        INSERT INTO augur_data.repo(repo_id, repo_group_id, repo_git, repo_path, repo_name, repo_added)
+        VALUES ({}, {}, {}, {}, {}, {})
+    """.format(repo_id, repo_group_id, '\''+repo['url']+'\'', '\'github.com/'+org+'/\'', '\''+repo['name']+'\'', '\''+repo['createdAt']+'\''))
+
+    augur_app.database.execute(insert_repo_sql)
+    return
+
+def get_value_from_fetch_data(data):
+    return data[0]
+
+def get_existing_repos(augur_app, repo_group_id):
+    repos_sql = s.sql.text("""
+            SELECT repo_name FROM repo
+            WHERE repo_group_id = '{}'
+        """.format(repo_group_id))
+
+    repos = augur_app.database.execute(repos_sql).fetchall()
+
+    if len(repos) == 0:
+        logger.info("Did not find any repositories stored in augur_database for repo_group_id {}\n".format(repo_group_id))
+
+    repos_list = list(map(get_value_from_fetch_data, repos))
+
+    return repos_list
+
+def get_org_id(augur_app, org):
```

Comment:
This exists so you can get the organization ID for which organization the repo belongs to. We don't make it easy to get this in any other way.

Reply:
(i.e., I think it would be confusing, and introduce breaking cases for people who used this feature otherwise.  For example, repo name alone does not need to be unique.